tests/integration-service/integration.go

```diff
@@ -442,6 +443,16 @@ func createComponent(f framework.Framework, testNamespace, applicationName, comp
 	err := f.AsKubeAdmin.CommonController.Github.CreateRef(componentRepoName, componentDefaultBranch, componentRevision, componentBaseBranchName)
 	Expect(err).ShouldNot(HaveOccurred())
 
+	var buildPipelineAnnotation map[string]string
+	customDockerBuildBundle := os.Getenv(constants.CUSTOM_DOCKER_BUILD_PIPELINE_BUNDLE_ENV)
+	if customDockerBuildBundle != "" {
+		buildPipelineAnnotation = map[string]string{
+			"build.appstudio.openshift.io/pipeline": fmt.Sprintf(`{"name":"docker-build", "bundle": "%s"}`, customDockerBuildBundle),
+		}
+	} else {
+		buildPipelineAnnotation = constants.DefaultDockerBuildPipelineBundle
+	}
+
```

Comment:
since this is used as-it-is everywhere, should we move it into a function instead to reduce the code duplication?

Reply:
Updated now.